src/openms/source/FORMAT/DATAACCESS/SiriusMzTabWriter.cpp

```diff
@@ -72,10 +72,10 @@ String SiriusMzTabWriter::extractFeatureId(const String& path)
   return feature_id;
 }
 
-std::map< String, Size > SiriusMzTabWriter::extract_columnname_to_columnindex(CsvFile& csvfile)
+std::map< std::string, Size > SiriusMzTabWriter::extract_columnname_to_columnindex(CsvFile& csvfile)
 {
   StringList header_row;
-  std::map< String, Size > columnname_to_columnindex;
+  std::map< std::string, Size > columnname_to_columnindex;
```

Comment:
I have been using String. Should I change them to std::string as well? I will.

Reply:
In practice not that important I guess. I try to use std::string whenever it is sufficient.

Reply:
Using `std::string` instead of `OpenMS::String` in the map is due to pyopenms wrapping. In general, you can use `OpenMS::String` without an issue:

```
Most complex STL constructs can be wrapped even if they are nested, however mixing them with user-defined types does not always work, see Limitations below. Nested std::vector constructs work well even with user-defined (OpenMS-defined) types. However, std::map<String, X> does not work (since String is user-defined, however, a primitive C++ type such as std::map<std::string, X> would work).
```
https://pyopenms.readthedocs.io/en/latest/wrap_classes.html